packages/components/Button/src/Button.styling.ts

```diff
@@ -15,19 +15,19 @@ import { defaultButtonTokens } from './ButtonTokens';
 
 export const buttonStates: (keyof ButtonTokens)[] = [
   'block',
-  'primary',
-  'subtle',
-  'outline',
-  'hovered',
   'small',
   'medium',
   'large',
   'hasContent',
   'hasIconAfter',
   'hasIconBefore',
+  'primary',
```

Comment:
This was moved because now primary has to take size into account when adjusting padding.